compiler/src/main/java/com/dylibso/chicory/compiler/internal/EmitterMap.java

```diff
@@ -19,6 +19,15 @@ final class EmitterMap {
                     .intrinsic(CompilerOpCode.CALL, Emitters::CALL)
                     .intrinsic(CompilerOpCode.CALL_INDIRECT, Emitters::CALL_INDIRECT)
 
+                    // ====== Exception Handling ======
+                    .intrinsic(CompilerOpCode.THROW, Emitters::THROW)
+                    .intrinsic(CompilerOpCode.THROW_REF, Emitters::THROW_REF)
+                    .intrinsic(CompilerOpCode.CATCH_COMPARE_TAG, Emitters::CATCH)
```

Comment:
better to align these names for consistency

Reply:
yes, just use the same names for Emitters and CompilerOpCode, see the inline suggestions